File: cvn-backend/user-service/src/main/java/com/cvn/user/config/RabbitMQConstants.java
package com.cvn.user.config;

public final class RabbitMQConstants {

    private RabbitMQConstants() {}

    /*
     * Exchange
     */
    public static final String CHILD_EXCHANGE =
            "child.exchange";

    /*
     * Queue
     */
    public static final String CHILD_REGISTERED_QUEUE =
            "child.registered.queue";

    /*
     * Routing Key
     */
    public static final String CHILD_REGISTERED_ROUTING_KEY =
            "child.registered";
}

File: cvn-backend/notification-service/notification-service/Config/RabbitMQConstants.cs
namespace notification_service.Config
{
    public static class RabbitMQConstants
    {
        public const string CHILD_EXCHANGE = "child.exchange";

        public const string CHILD_REGISTERED_QUEUE = "child.registered.notification.queue";

        public const string CHILD_REGISTERED_ROUTING_KEY = "child.registered";
    }
}